.github/workflows/retrofw_release.yml

```diff
@@ -16,19 +16,21 @@ jobs:
     - name: Create Build Environment
       run: >
         sudo apt update &&
-        sudo apt install -y curl git smpq &&
-        curl -L https://github.com/retrofw/buildroot/releases/download/2018.02.11/mipsel-linux-uclibc_x64.tar.gz -o retrofw-toolchain.tar.xz &&
-        sudo mkdir -p /opt/mipsel-linux-uclibc && sudo chown -R "${USER}:" /opt/mipsel-linux-uclibc &&
-        tar -C /opt -xvf retrofw-toolchain.tar.xz
+        sudo apt install -y curl cmake git smpq squashfs-tools gettext &&
```

Comment:
squashfs ... so a good opportunity to get rid of the mpq here?

Reply:
Yeah possibly but need to test the compression